devel/python/python/ert_gui/ertwidgets/models/selectable_list_model.py

```diff
@@ -0,0 +1,46 @@
+from PyQt4.QtCore import QObject, pyqtSignal
+
+
+class SelectableListModel(QObject):
+    modelChanged = pyqtSignal()
+    selectionChanged = pyqtSignal()
+
+    def __init__(self, items):
+        QObject.__init__(self)
+        self._selection = {}
+        self._items = items
+
+    def getList(self):
+        return self._items
+
+    def isValueSelected(self, value):
+        if self._selection.has_key(value):
+            return self._selection[value]
+        else:
+            return True
```

Comment:
Can't you use `self._selection.get(value, True)`, which returns the second argument if `value not in self._selection`?